src/pages/stats/[login].tsx

```diff
@@ -225,6 +241,16 @@ export default function Stats() {
               onChange={(e) => setSearchQuery(e.target.value)}
             />
           </div>
+          <div className="flex sm:items-start items-center">
```

Comment:
Here `flex-direction` is `row` (by default), you should probably use the `justify` property.

![image](https://github.com/DevLeonardoCommunity/github-stats/assets/7253929/de7ba142-f7ac-40ad-a90a-a744c79c3725)

Reply:
are you seeing them centered?
I see them all left-aligned
![Screenshot 2024-01-26 at 09 13 04](https://github.com/DevLeonardoCommunity/github-stats/assets/89919203/bd2f93d4-ad20-43c6-b623-953287fce187)

Reply:
They get centered on mobile/smaller screens